molgenis-dataexplorer/src/main/resources/js/dataexplorer-data.js

```diff
@@ -92,7 +92,8 @@
 			query : {
 				rules : [entityQuery.q]
 			},
-			colNames : $('input[name=colNames]:checked').val()
+			colNames : $('input[name=colNames]:checked').val(),
+			downloadType : $('input[name=downloadTypes]:checked').val()
 		};
```

Comment:
Does this make sense in the case of a galaxy export?
And in the case of excel, what is supposed to happen when the attribute labels are chosen as column names?